### PR TITLE
Use f-strings instead of format or old-style %

### PR DESCRIPTION
**Pull request by mulhod**

### CODE REVIEW

**Comment by mulhod** [comment]
According to the code coverage diff, coverage increases very slightly by 0.01%. So, why `codecov` is saying differently I'm not really sure.

**Comment by desilinguist** [comment]
@mulhod can you push another commit?

**Comment by mulhod** [comment]
Hmm, `codecov` check still failed.

**Comment by mulhod** [comment]
Or perhaps I'm just reading it wrong? The `codecov/patch` check failed. But not `codecov/project`.

**Comment by desilinguist** [comment]
Yeah, that's the patch diff that failed – which checks whether any new lines you added are also covered by tests. I am guessing these are because you broke some long strings into smaller ones?

**Comment by mulhod** [comment]
I did break some lines up, but it appears the total didn't check by much. Maybe 2 lines more?

Anyway, it's a little unclear to me how this can be addressed or if it should.

**Comment by desilinguist** [comment]
I don't think we need to worry about it.

**Review by desilinguist — APPROVED** [review]
LGTM 👍🏽 !

**Review by bndgyawali — APPROVED** [review]
I have one small suggestion, else it LGTM.

File: skll/data/readers.py [hunk 1]
```diff
@@ -280,7 +280,7 @@ def feat_dict_generator():
                 for ex_num, (_, _, feat_dict) in enumerate(self._sub_read(f)):
                     yield feat_dict
                     if ex_num % 100 == 0:
-                        self._print_progress('{100 * (ex_num / total):.8}%')
+                        self._print_progress('{100 * ex_num / total:.8}%')
```
Comment (desilinguist) [inline skll/data/readers.py:283]: Does this actually work?

Comment (mulhod) [reply skll/data/readers.py:283]: Yes, it does appear to.

**Pull request merged**